controllers/hvpa_controller.go

```diff
@@ -844,6 +844,13 @@ func getWeightedRequests(vpaStatus *vpa_api.VerticalPodAutoscalerStatus, hvpa *a
 		return nil, false, nil, nil
 	}
 	for k, v := range vpaStatus.Conditions {
+		if v.Type == vpa_api.ConfigUnsupported || v.Type == vpa_api.ConfigDeprecated || v.Type == vpa_api.LowConfidence {
```

Comment:
A suggestion for readability. Why not put the condition types in an array and loop over them? The code would be more easier to understand and extend in case more conditions are applicable later, WDYT?

Reply:
sure. sounds good. will do.
thanks.

Reply:
Done in https://github.com/gardener/hvpa-controller/pull/69/commits/8913e48fc948e79ce3ca0ba51e596cc915eb1ea6